_field-types/mapping-parameters/analyzer.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Analyzer
+Parent: Mapping parameters
+Grand_parent: Mapping and field types
+nav_order: 5
+has_children: false
+has_toc: false
+---
+
+# Analyzer
+
+The `analyzer` mapping parameter is used to define the text analysis process that is applied to a text field during both indexing and searching operations.
+
+The key functions of the `analyzer` mapping parameter are:
+
+1. **Tokenization:** The analyzer determines how the text is broken down into individual tokens (words, numbers) that can be indexed and searched.
+
+2. **Normalization:** The analyzer can apply various normalization techniques, such as converting text to lowercase, removing stopwords, and stemming/lemmatizing words.
+
+3. **Consistency:** By defining the same analyzer for both indexing and searching, you ensure that the text analysis process is consistent, which helps improve the relevance of search results.
+
+4. **Customization:** OpenSearch allows you to define custom analyzers by specifying the tokenizer, character filters, and token filters to use. This gives you fine-grained control over the text analysis process.
+
+------------
+
+## Example
```

Comment:
@mgodwan Is this an appropriate example relevant to a developer using OpenSearch?

Reply:
Yes, this looks good